src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -42,13 +43,15 @@
   private String logFile = null;
   private HdfsSinkConnectorConfig conf = null;
   private HdfsStorage storage = null;
+  private UUID logFileId;
```

Comment:
this now seems unused

Reply:
I think my comment remains. Why do we need a UUID?

Reply:
In some recent logs we saw that a WAL lease was created by one UUID but closed by 3 different UUIDs + the original UUID, it would help us investigate if any potential issues arise.

Reply:
A second use would be for the case if two connectors with the same name are started, then the UUID would be the only differentiator in those logs.

Reply:
We could also opt for a more compact random number if that's preferable

Reply:
The problem with a random part is that we can't map it back to a specific task. 

I don't there's a strong case for adding it. The task ID should suffice to identify the task that issues the log messages. 
Finally, the use case of running multiple connect clusters with the connectors that have the same name across cluster is fairly uncommon to support it here IMO.

Reply:
Updated PR